src/payers/hvoice_payer.cpp

```diff
@@ -20,6 +20,12 @@ namespace hypha
             std::make_tuple(recipient, quantity, memo))
             .send();
 
+        eosio::action(
+                eosio::permission_level{m_dao.get_self(), eosio::name("active")},
+                m_dao.getSettingOrFail<name>(HYPHA_TOKEN_HVOICE_CONTRACT), eosio::name("issue"),
+                std::make_tuple(recipient, quantity, memo))
+                .send();
+
```

Comment:
I think that ```issue``` only allows issuance to the issuer account (which should be dao.hypha), and then you'd need to also call ```transfer```.  Check what the hypha_payer.cpp looks like and it should be very similar.

Reply:
I removed the transfer action. Adding it back.
I suppose we want to prevent from users other than the issuer to transfer & retire. Is that correct?

Reply:
Updated to use the `issueToken` and based on integration branch.